### DISCUSSION TITLE
Make erect penises noticable by others through clothing

### DISCUSSION THREAD

**Discussion by Gwendeadlyn** [littlesera/LSCG#467] [Ideas]
Something like the messages only subs get based on some factors for example (idk, you know better than me Sera)

Scenarios: 
-someone sitting in lap
-someone rubbing bodies
-someone caressing it
-kiss penis
-slap penis
-kick penis
-scratch penis

Optional:
-head in lap

Anything else you may think of that I missed

**Comment by littlesera** [comment]
Implemented in v0.3.45
